apps/Standalone/src/designer/app/AzureLogicAppsDesigner/DesignerCommandBar.tsx

```diff
@@ -95,7 +95,7 @@ export const DesignerCommandBar = ({
     const customCodeFilesWithData = getCustomCodeFilesWithData(designerState.customCode);
 
     if (!hasParametersErrors) {
-      await saveWorkflow(serializedWorkflow, customCodeFilesWithData, () => dispatch(resetDesignerDirtyState()));
+      await saveWorkflow(serializedWorkflow, customCodeFilesWithData, () => dispatch(resetDesignerDirtyState(undefined)));
```

Comment:
Please add the argument name here, it is confusing on which argument is set as undefined and why do we have to set it undefined, shouldnt the argument assume when not sent it is undefined

Reply:
Its because the thunk was changed to be async (_: unknown, thunkAPI: any) .There isn't an actual argument we should be providing I dont think